### PR TITLE
Image zoom

### PR DESCRIPTION
**Pull request by bhlynsky**

### CODE REVIEW

File: docs/website/src/md-image/md-image.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    originalImage.setAttribute('mode', 'fit');
+    return originalImage;
+  }
+
+  @memoize()
+  get $closeButton() {
+    const button = document.createElement('button');
+    button.onclick = this.onClose.bind(this);
+    button.setAttribute('class', 'close-button');
+    button.innerText = 'X';
```
Comment (ala-n) [inline docs/website/src/md-image/md-image.ts:35]: Cosmetic: most likely `x` visual representative will depend on the browser's default font, so consider something more static

File: docs/website/src/md-image/md-image.ts [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    button.style.display = 'none';
```
Comment (ala-n) [inline docs/website/src/md-image/md-image.ts:36]: Do we need to handle it with inline styles, we can use attributes/classes of container to handle button visibility from css.

File: docs/website/src/md-image/md-image.ts [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    if (!this.inZoom) return;
+    this.inZoom = false;
+
+    this.$image.style.removeProperty('transform');
+    this.style.width = 'auto';
+    this.style.height = 'auto';
+
+    this.$closeButton.style.display = 'none';
+
+    this.$$off(this.onPointerDown);
```
Comment (ala-n) [inline docs/website/src/md-image/md-image.ts:70]: I'd recommend to create a method `reset` / `cancelZoom` or something like that to separate everything that relates to event handling from exact action

File: docs/website/.editorconfig [hunk 4]
```diff
@@ -9,6 +9,8 @@ root = true
 indent_style = space
 indent_size = 2
 ij_any_blank_lines_around_method = 0
+quote_type = single
```
Comment (ala-n) [inline docs/website/.editorconfig:12]: Awesome 👍🏻

**Review by ala-n — APPROVED** [review]

**Review by NastaLeo — APPROVED** [review]

File: docs/website/src/eak-zoom-image/eak-zoom-image.ts [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+import { boolAttr } from '@exadel/esl/modules/esl-utils/decorators/bool-attr';
+import { memoize } from '@exadel/esl/modules/esl-utils/decorators/memoize';
+import { getTouchPoint } from '@exadel/esl/modules/esl-utils/dom/events/misc';
+
+import type { Point } from '@exadel/esl/modules/esl-utils/dom/events/misc';
+
+export class EAKZoomImage extends ESLBaseElement {
+  static is = 'eak-zoom-image';
+
+  @attr({ dataAttr: true })
```
Comment (abarmina) [inline docs/website/src/eak-zoom-image/eak-zoom-image.ts:13]: for better readability here and below, mb will we write it in one line?
```suggestion
  @attr({ dataAttr: true }) public src: string;
```

**Review by abarmina — APPROVED** [review]

File: docs/website/src/site.ts [hunk 6]
```diff
@@ -20,3 +23,5 @@ ESLPanelGroup.register();
 ESLPanel.register();
 ESLTrigger.register();
 ESLScrollbar.register();
+ESLImage.register();
+EAKZoomImage.register();
```
[the remark this answers is not in the document]
Comment (ala-n) [reply docs/website/src/site.ts:27]: +1 or if some order requiroments, it sounds logical to move all EAK components after the base ESL components, but keep them in groups.

**Pull request merged**